### PR TITLE
Template strings in MainTemplate file

### PR DESCRIPTION
**Pull request by jescalan**
Just a cosmetic change here, starting small. Hope to be able to make larger contributions down the line! No breaking here, no tests needed.

### CODE REVIEW

File: lib/MainTemplate.js [hunk 1]
```diff
@@ -30,8 +30,7 @@ module.exports = class MainTemplate extends Template {
 			const buf = [];
 			if(chunk.entryModule) {
 				buf.push("// Load entry module and return exports");
-				buf.push("return " + this.renderRequireFunctionForModule(hash, chunk, JSON.stringify(chunk.entryModule.id)) +
-					"(" + this.requireFn + ".s = " + JSON.stringify(chunk.entryModule.id) + ");");
+				buf.push(`return ${this.renderRequireFunctionForModule(hash, chunk, JSON.stringify(chunk.entryModule.id))}(${this.requireFn}.s = "${JSON.stringify(chunk.entryModule.id)}");`);
```
Comment (timse) [inline lib/MainTemplate.js]: `"${JSON.stringify(chunk.entryModule.id)}"`

i think these `"` should be omitted

**Comment by timse** [comment]
also i think its appreciated if you stick to the provided template :)

File: lib/MainTemplate.js [hunk 2]
```diff
@@ -173,14 +172,14 @@ module.exports = class MainTemplate extends Template {
...
-		buf.push("function " + this.requireFn + "(moduleId) {");
+		buf.push(`function ${this.requireFn}(moduleId) {`);
 		buf.push(this.indent(this.applyPluginsWaterfall("require", "", chunk, hash)));
 		buf.push("}");
 		buf.push("");
 		buf.push(this.asString(this.applyPluginsWaterfall("require-extensions", "", chunk, hash)));
 		buf.push("");
 		buf.push(this.asString(this.applyPluginsWaterfall("startup", "", chunk, hash)));
-		let source = this.applyPluginsWaterfall("render", new OriginalSource(this.prefix(buf, " \t") + "\n", "webpack/bootstrap " + hash), chunk, hash, moduleTemplate, dependencyTemplates);
+		let source = this.applyPluginsWaterfall("render", new OriginalSource(this.prefix(buf, " \t") + `\n`, `webpack/bootstrap ${hash}`), chunk, hash, moduleTemplate, dependencyTemplates);
```
Comment (sokra) [inline lib/MainTemplate.js]: `` `\n`,`` -> ` "\n",`

File: lib/MainTemplate.js (hunk 1 above)
Comment (sokra) [inline lib/MainTemplate.js]: yep

File: lib/MainTemplate.js [hunk 3]
```diff
@@ -194,7 +193,7 @@ module.exports = class MainTemplate extends Template {
 	}
 
 	renderAddModule(hash, chunk, varModuleId, varModule) {
-		return this.applyPluginsWaterfall("add-module", "modules[" + varModuleId + "] = " + varModule + ";", chunk, hash, varModuleId, varModule);
+		return this.applyPluginsWaterfall("add-module", `modules["${varModuleId}"] = ${varModule};`, chunk, hash, varModuleId, varModule);
```
Comment (sokra) [inline lib/MainTemplate.js]: Here the same. `"` must be omited. `modules["${varModuleId}"]` -> `modules[${varModuleId}]`

**Comment by jescalan** [comment]
Ok @sokra, sorry for the delay but just took care of the pieces you commented on!

File: lib/MainTemplate.js [hunk 4]
```diff
@@ -71,7 +70,7 @@ module.exports = class MainTemplate extends Template {
 					"var threw = true;",
 					"try {",
 					this.indent([
-						"modules[moduleId].call(module.exports, module, module.exports, " + this.renderRequireFunctionForModule(hash, chunk, "moduleId") + ");",
+						`modules[moduleId].call(module.exports, module, module.exports, ${this.renderRequireFunctionForModule(hash, chunk, "moduleId")}");`,
```
Comment (sokra) [inline lib/MainTemplate.js]: The last `"` must be removed.

**Comment by jescalan** [comment]
Removed the extra quote. Sorry about that!

**Comment by jescalan** [comment]
ping @sokra 😁

**Comment by sokra** [comment]
Thanks

**Pull request merged**